script.js

```diff
@@ -1,4 +1,235 @@
+    returnString = returnString + ',' + newCard;
+  }
+  return returnString;
+};
+
+//deal cards to hands
+var dealCardToHand = function () {
+  for (var i = 0; i < 2; i ++) {
+    hand.push(shuffledDeck.pop())
+  }
+  return hand;
+};
+
+//=======================Store player and computer's cards=====================
+var playerHand = [];
+var computerHand = [];
+let maxCardLimit = 21; //max value before bust is 21
+let dealerMinValue = 16; // dealer has to minimally deal 16
+let numOfCards = 0;
+//=======================Main function=====================
+var main = function (input){
+  var myOutputValue = '';
+  currentGameMode = GAME_MODE_DEAL_CARDS
+  var genericOutput = `Your cards: ${printCards(playerHand)} <br>Computer's cards: ${printCards(computerHand)}.`
+  // draw one new card for player and computer each
+  playerHand.push(shuffledDeck.pop())
+  computerHand.push(shuffledDeck.pop())
+  // generate another card for each player
+  playerHand.push(shuffledDeck.pop())
+  computerHand.push(shuffledDeck.pop())
+  if (playerHand.length == 2){ 
+    currentGameMode == GAME_MODE_HIT_OR_STAND
+    console.log(`game mode: ${currentGameMode}`)
+    console.log (`player hand ${printCards(playerHand)}, com hand: ${printCards(computerHand)}`)
+    myOutputValue = `${genericOutput}<br>
+    Please type 'hit' or 'stand' to decide if you wish to draw cards or end your turn.`
+    }
+//=======================Determine if player hits or stands=====================
+if (input.toLowerCase().includes('hit')){
+  currentGameMode = GAME_MODE_HIT_OR_STAND
+  currentPlayer = PLAYER
+  console.log(`input: ${input}, game mode: ${currentGameMode}`)
+  // draw one card for player and computer each
+  playerHand = dealCardToHand(deck)
+  // add new card to player's hand array
+  playerHand.push(deck.pop())
+  if (cardTotalSum > maxCardLimit){
+  gameOver = true;
+  myOutputValue = `${genericOutput}<br>Player went bust! Hit refresh to start again.`
+  }
+  else if (cardTotalSum < maxCardLimit)
+  myOutputValue = `${genericOutput}<br>Please type 'hit' or 'stand' to decide if you wish to draw cards or end your turn.`
+  }
+if (input.toLowerCase().includes('stand')){
+  currentGameMode = GAME_MODE_HIT_OR_STAND
+  console.log(`input: ${input}, game mode: ${currentGameMode}`)
+  currentPlayer = COMPUTER
+  myOutputValue = `${genericOutput}<br>Player chose to stand. Press submit for Computer's turn.`
+}
+return myOutputValue;
+}
+
+//=======================Determine winner=====================
+var winner = function (){
+  if (playerCard.rank > computerCard.rank
+    && input.toLowerCase().includes('stand')){
+    myOutputValue = `You win! Hit refresh to play again.`
+  } 
+  if (computerCard.rank > playerCard.rank
+    && input.toLowerCase().includes('stand')){
+    myOutputValue = `You lose! Hit refresh to play again.`
+  } 
+
+}
+
+//=======================Calculate sum total of cards=====================
+var calcHandTotal = function (hand){
+  var cardTotalSum = 0;
```

Comment:
You have to declare the `cardTotalSum` outside this function if you want to use it outside the function. (Eg. you used `cardTotalSum`  in line 138)